kokoro/scripts/test/go_test.sh

```diff
@@ -136,17 +139,18 @@ function set_zones() {
   # T2A machines are only available on us-central1-{a,b,f}.
   # See warning above about changing regions.
   elif [[ "${ARCH:-}" == "aarch64" ]]; then
-    zones=(
+    zone_list=(
       us-central1-a
       us-central1-b
       us-central1-f
     )
   else
-    zones=(
+    zone_list=(
       invalid_zone
     )
   fi
-  export ZONES="$(join_by , ${zones[@]})"
+  zones=$(join_by , "${zone_list[@]}")
```

Comment:
I was thinking

```
ZONES="$(join_by , "${zone_list[@]}")"
export ZONES
```

but what you have is OK too.

Note that I added an extra pair of quotes (in case the joined zones list somehow had a space), which is a best practice and somehow works fine and doesn't confuse bash, yay.

Reply:
Gosh but it does confuse me.